packages/react-components/react-positioning/src/utils/writeContainerupdates.ts

```diff
@@ -50,8 +51,17 @@ export function writeContainerUpdates(options: {
   const x = Math.round(coordinates.x * devicePixelRatio) / devicePixelRatio;
   const y = Math.round(coordinates.y * devicePixelRatio) / devicePixelRatio;
 
+  if (useTransform) {
+    Object.assign(container.style, {
+      transform: lowPPI ? `translate(${x}px, ${y}px)` : `translate3d(${x}px, ${y}px, 0)`,
+      position: strategy,
```

Comment:
nit, since the strategy will be the same regardless of `useTransform`, I'd assign it before the rest

Reply:
Updated to assign `strategy` before the if condition. Thank you!